src/pybind/mgr/dashboard/controllers/osd.py

```diff
@@ -184,6 +195,19 @@ def create(self, uuid=None, svc_id=None):
             'uuid': uuid,
         }
 
+    @CreatePermission
+    @raise_if_no_orchestrator
+    @handle_orchestrator_error('osd')
+    @osd_task('create', {'dg_names': '{dg_names}'})
+    def create(self, drive_groups, dg_names):
+        # dg_names is required in the body for tracking tasks
+        _ = dg_names
+        orch = OrchClient.instance()
+        try:
+            orch.osds.create(DriveGroupSpecs(drive_groups).drive_groups)
+        except (ValueError, TypeError, DriveGroupValidationError) as e:
+            raise DashboardException(e, component='osd')
+
```

Comment:
As stated above: no need for `/create`. `POST <resource_uri>` is the way for creating new resources in a REST API.